package.json

```
@@ -4,6 +4,7 @@
   "sideEffects": false,
   "version": "2.12.0",
   "module": "./dist/es/src/index.js",
+  "types": "./dist/es/src/index.d.ts",
```

```suggestion
  "types": "./dist/src/index.d.ts",
```
That's the way we do in other libraries